.travis.yml

```diff
@@ -38,7 +38,7 @@ matrix:
 
     - language: node_js
       node_js:
-        - '8'
+        - '10'
```

Comment:
Why is the target set on node 10 ?

Reply:
Because `npm audit` is a node 10 feature